tools/integration-tests/tester/framework/config/flags.go

```diff
@@ -47,11 +49,22 @@ func (s GoShimmer) CreateFlags() []string {
 	flags = append(
 		[]string{
 			"--node.enablePlugins=Webapi tools Endpoint",
-			fmt.Sprintf("--autopeering.seed=base58:%s", base58.Encode(s.Seed)),
 			fmt.Sprintf("--node.enablePlugins=%s", setToString(enabledPlugins)),
 			fmt.Sprintf("--node.disablePlugins=%s", setToString(disabledPlugins)),
 		},
 		flags...)
+
+	// manually add seed to flags if autopeering is disabled
```

Comment:
This is super ugly, the seed has nothing to do with the autopeering anymore! However unless we rename this as part of #1507 I guess there is no way around it.

Reply:
Can we maybe add a comment explaining that this is only because the seed is in the wrong struct?

Reply:
Done. Yeah it's ugly and should be removed once we do #1507.